src/main/java/eu/europa/ec/fisheries/uvms/exchange/model/mapper/ExchangeModuleRequestMapper.java

```diff
@@ -454,8 +454,6 @@ private static void populateBaseProperties(ExchangeBaseRequest request, String f
         request.setDate(date);
         request.setMessageGuid(messageGuid);
         request.setPluginType(pluginType);
-        request.setTodt(todt);
```

Comment:
This method is called both for incoming and outgoing messages. For incoming it is OK to keep the attributes. So, create another one (e.g. `populateBasePropertiesForOutgoing()`) that does not populate the `to` and `toDt` fields. Call the new method from `createSendFaQueryMessageRequest` and `createSendFaReportMessageRequest`.